test/TEST-29-UDEV-ID_RENAMING/testsuite.sh

```diff
@@ -21,19 +21,19 @@ EOF
 udevadm control --log-priority=debug --reload
 udevadm trigger --action=add --settle /sys/devices/virtual/net/lo
 udevadm info /sys/devices/virtual/net/lo
-
+sleep 1
```

Comment:
IIUC, `udevadm --settle` waits for udev, but it does not wait for systemd, which is asynchronous, so we can get wrong state from systemctl. Sleeping here seems a reasonable workaround.

Reply:
Yeah, I think so.